src/pages/Task.tsx

```diff
@@ -54,9 +59,20 @@ export const Task: React.FC = () => {
   } = data;
 
   return (
-    <>
+    <PageWrapper>
       <BreadCrumb displayName={displayName} version={version} isTask={true} />
-      {tab === Tab.Tests && <TestsTable />}
-    </>
+      <PageHeader>
```

Comment:
Could we move this into the sidebar? In general the task name will be short, which will leave a fair amount of whitespace that could be used for something useful. If the task name is long, it's fine to truncate it

Reply:
going with designs! talk to ben for changes :)